todo_app/app.py

```diff
@@ -1,34 +1,107 @@
 import os
+import requests
 from todo_app.data.mongodb import MongoDb
+from todo_app.data.user import User
 from flask.globals import request
-from flask import Flask, render_template, redirect
+from flask import Flask, render_template, redirect, session, abort
+from flask_login import LoginManager
+from flask_login import login_required
+from flask_login import login_user
+from flask_login import current_user
+
+from oauthlib.oauth2 import WebApplicationClient
 
 from todo_app.view_models.view_model import ViewModel 
 
 def create_app():
     app = Flask(__name__, template_folder="templates")
 
+    disable_login = os.getenv('LOGIN_DISABLED')
+    if disable_login:
+      print("Github authentication disabled")
+      app.config['LOGIN_DISABLED'] = True      
+
+    app.config['SECRET_KEY'] = os.getenv('SECRET_KEY')
+    github_client_id = os.getenv('GITHIB_CLIENT_ID')
```

Comment:
Minor typo that threw me when trying to set up the environment variables!
```suggestion
    github_client_id = os.getenv('GITHUB_CLIENT_ID')
```